_commands/toys/cowsay.md

```diff
@@ -3,22 +3,102 @@
 
 cowsay
 -------
+`cowsay` generates an ASCII picture of a cow saying something provided by  the  user.
 
-ToDo: Needs Completion
 
 ~~~ bash
-*short* example of command usage and output
+$ cowsay Hi There
+ __________ 
+< Hi There >
+ ---------- 
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
 ~~~
 
 <!--more-->
 
+
+
 ### Useful Options / Examples
+There are many useful options for this intersting cowsay command.
+The complete list is: cowsay  [-e  eye_string] [-f cowfile] [-h] [-l] [-n] [-T tongue_string] [-W column] [-bdgpstwy]
 
-#### Example command
+-e allows the user to customize the cow's eyes which are defaulted to "oo" as seen above.
+-T allows the user to add a tongue string to the cow image. By default there is no tongue string.
+-W allows the user to customize the number of character printed in the message before the message wraps around
+-[-bdgpstwy] are all options to define different pre-made cows emotional states. For example, adding the -g flag will result in a greedy cow to print and -s will result in a stoned cow to print.
 
+####cowsay [-bdgpstwy] [`user_input`]
+~~~ bash
+$ cowsay -s hi there
+ __________ 
+< hi there >
+ ---------- 
+        \   ^__^
+         \  (**)\_______
+            (__)\       )\/\
+             U  ||----w |
+                ||     ||
+~~~
 ##### Break it down
+There are several provided modes which change the appearance of the cow depending on its particular emotional/physical state
+The user can set this state using these flags
+####cowsay [-f cowfile] [`user_string`]
+~~~ bash
+$ cowsay -f tux hi there
+ __________ 
+< hi there >
+ ---------- 
+   \
+    \
+        .--.
+       |o_o |
+       |:_/ |
+      //   \ \
+     (|     | )
+    /'\_   _/`\
+    \___)=(___/
 
-#### Example command
+~~~
 
 ##### Break it down
+The -f flag allows for other ascii images to be printed
+These images are pulled from .cow files, some of which come preinstalled and the full list can be found by typing:
+~~~ bash
+$ cowsay -l
+Cow files in /usr/local/Cellar/cowsay/3.03/share/cows:
+beavis.zen bong bud-frogs bunny cheese cower daemon default dragon
+dragon-and-cow elephant elephant-in-snake eyes flaming-sheep ghostbusters
+head-in hellokitty kiss kitty koala kosh luke-koala meow milk moofasa moose
+mutilated ren satanic sheep skeleton small sodomized stegosaurus stimpy
+supermilker surgery telebears three-eyes turkey turtle tux udder vader
+vader-koala www
+~~~
 
```

Comment:
![screen shot 2016-03-20 at 10 03 55 pm](https://cloud.githubusercontent.com/assets/7095187/13909344/bfd381e6-eee7-11e5-826c-a88bbf0a542d.png)
I think you meant to put the `~~~` on line 65 somewhere else. The formatting below the top is a little off.